integration_test/support/mix_task_runner.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Phoenix.Integration.MixTaskRunner do
```

Comment:
It probably doesn't need to be a separate module anymore and we can move this helper function into code_generator_case?